Dockerfile

```diff
@@ -1,56 +1 @@
-FROM drydock-prod.workiva.net/workiva/dart_build_image:1
```

Comment:
I don't think we can remove all of this - afaik, we at least need a stage that runs `pub get` so that workiva-build can scrape the `pubspec.lock` for auditing and dep tracking purposes